components/event-sources/Makefile

```diff
@@ -53,13 +56,16 @@ vendor: Gopkg.lock
 	@echo '+ Pulling vendored dependencies'
 	@dep ensure -v --vendor-only
 
+$(patched_trace): hack/set-span-id.patch vendor
+	@echo '+ Patching opencensus dependency'
+	@git apply ${REPO_ROOT_DIR}/components/event-sources/hack/set-span-id.patch
```

Comment:
`patch -p0 < hack/set-span-id.patch` would be preferable here. Your file is not checked out due to the ignored vendor directory, so you `git apply` a file unknown to Git. It may work, but it feels a bit clunky to me.